internal/gonut/cf/cf.go

```diff
@@ -289,6 +304,27 @@ func getApp(appName string) (*AppDetails, error) {
 	return cfCurlAppByGUID(appGUID)
 }
 
+// getAppRoute returns the public URL of the application
+// using its name and the Cloud Foundry host domain.
+func getAppRoute(appName string) (string, error) {
+	domain, err := getDomain(appName)
+	if err != nil {
+		return "", err
+	}
+
+	return fmt.Sprintf("http://%s.%s", appName, domain), nil
+}
+
+// getAppStatusCode sends a GET request to the given URL
+// and returns the statuscode.
+func getAppStatusCode(appRoute string) (int, error) {
```

Comment:
I would like to suggest one more thing here: It is not just the HTTP response code that is interesting, it would also be interesting to notify the whole response time. Could you add a third return value, which is the overall time it took to get a result?